contracts/UFragments.sol

```diff
@@ -175,8 +137,8 @@ contract UFragments is ERC20Detailed, Ownable {
         ERC20Detailed.initialize("Ampleforth", "AMPL", uint8(DECIMALS));
         Ownable.initialize(owner_);
 
-        rebasePaused = false;
-        tokenPaused = false;
+        rebasePausedDeprecated = false;
+        tokenPausedDeprecated = false;
```

Comment:
the initialize function should be removed entirely when deploying a new template (since it is never called)

Reply:
Good to know.

There could be a future where we do a separate non-upgrade deployment. Could be on Ethereum or an alternate chain that works with solidity tools. In that case, it'd be useful to have it there already.

Reply:
I see - in that case I would recommend keeping as a comment since you want to avoid deploying unreachable code